skrifa/src/color/mod.rs

```diff
@@ -263,7 +263,10 @@ pub trait ColorPainter {
     /// Open a new layer, and merge the layer down using `composite_mode` when
     /// [`pop_layer`](ColorPainter::pop_layer) is called, signalling that this layer is done drawing.
     fn push_layer(&mut self, composite_mode: CompositeMode);
-    fn pop_layer(&mut self);
+    fn pop_layer(&mut self) {}
+    fn pop_layer_with_mode(&mut self, _composite_mode: CompositeMode) {
+        self.pop_layer();
+    }
```

Comment:
This part looks right. Just need to update all of the call sites to use the new method.

Would also be nice to have a short doc comment mentioning why this is useful and that only one of `pop_layer` or `pop_layer_with_mode` should be implemented.

Reply:
> This part looks right. Just need to update all of the call sites to use the new method.

What do you have in mind? traverse calling the new one? NullPen etc should be fine I assume.

Reply:
I believe it's just the two `pop_layer` calls here that need to be changed to `pop_layer_with_mode`: https://github.com/googlefonts/fontations/blob/c32c0a6442371d0206f82a4a981b3854f313aaae/skrifa/src/color/traversal.rs#L590

Reply:
Ah, right. Fixed.